imports/lib/models/base.ts

```diff
@@ -167,9 +166,12 @@ class Base<T extends BaseType> extends Mongo.Collection<T> {
           return [];
         }
 
-        let query = q;
+        let query: Mongo.Selector<T> | undefined = q;
         if (modifier) {
-          query = modifier.apply(this, [q]);
+          query = modifier(this.userId, q);
```

Comment:
Is there a reason to prefer `this.userId` over just capturing `userId` from the first argument to `publish()` here?

Reply:
`userId` is the first argument to `modifier`, which is a functional argument to publish.